projects/igniteui-angular/src/lib/grids/filtering/grid-filtering-row.component.ts

```diff
@@ -386,6 +391,8 @@ export class IgxGridFilteringRowComponent implements AfterViewInit {
      */
     public clearInput() {
         this.value = null;
+        const selectedChip = this.expressionsList.findIndex(ex => ex.isSelected === true);
```

Comment:
Rename `selectedChip` to `selectedIndex`. Should we always delete the selected item when clearing the input?

Reply:
That is fixing the issue with the empty chip - when click the X input button.

Reply:
Addressed